lib/castle/client_actions/list_items.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+module Castle
+  module ClientActions
+    module ListItems
+      def archive_list_item(options)
```

Comment:
Sorry, I've missed that on the first round, but shouldn't these methods correspond to the interface of the caller underneath? E.g. for `Castle::API::ListItems::Archive.call` `options` argument is optional, since there is a default. I think the same should be true for the ClientActions methods, even though some of them will fail the validation later on.
```suggestion
      def archive_list_item(options = {})
```